report/renderer/dom.js

```diff
@@ -67,7 +70,7 @@ export class DOM {
   createElementNS(namespaceURI, name, className, attrs = {}) {
     const element = this._document.createElementNS(namespaceURI, name);
     if (className) {
-      element.className = className;
+      for (const token of className.split(' ')) element.classList.add(token);
```

Comment:
what necessitated this?

Reply:
Svg element class list can be set via className.

Reply:
> Svg element class list can't be set via className.

Whoops. Tell me we don't have tests for something without telling me we don't have tests for something :) Can you drop in a test at the same time?

Reply:
also, `element.classList.add()` throws on the empty string and assorted other characters, so need to handle at least the whitespace corner cases, `' '`, `'name '`, etc. We shouldn't be taking class names from user input so it doesn't need to be bulletproof, but we don't want an errant space to take down the renderer (vs for whatever reason the html gods of the 90s in their wisdom decreed `element.className = 'asdf\n'` should work fine)